model/notification.go

```diff
@@ -0,0 +1,229 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+import (
+	"strings"
+	"unicode"
+	"unicode/utf8"
+
+	"github.com/mattermost/mattermost-server/v5/utils/markdown"
+)
+
+type ExplicitMentions struct {
+	// Mentions contains the ID of each user that was mentioned and how they were mentioned.
+	Mentions map[string]MentionType
+
+	// OtherPotentialMentions contains a list of strings that looked like mentions, but didn't have
+	// a corresponding keyword.
+	OtherPotentialMentions []string
+
+	// HereMentioned is true if the message contained @here.
```

Comment:
Is there any use case for overwriting this property?

Reply:
This is a good question but I'm not the author of this code, I'll try to look at the source PR for this change.

Reply:
Just from looking at the code I can see that only `model` is using this property. Maybe a getter is enough for plugins? Then it doesn't have to be exported.

Reply:
`HereMentioned`is used in [app/notification.go](https://github.com/mattermost/mattermost-server/blob/3e3fcdf84502ac0a4652e1d6e0459e67ed5ac188/app/notification.go#L177) as well

Reply:
But only to read the variable. A getter would be enough for `app`.

Reply:
It's not a big change actually, I'll fix it in the next commit.